rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,170 @@
+
+In accordance with RFC 102, the moderation team uses the NixOS foundation mission statement as a guideline. However, the interpretation of this mission statement and the subsequent [Code of Conduct][coc] is largely at the team's discretion. RFC 102 acknowledged the absence of explicit guidance on this matter.
+
+We contend that this ambiguity has allowed the moderation team to shape community values without consent, either direct or implied. The exercise of power in this manner, to suppress reasonable dissent and foster an illusion of unity, has a chilling effect that hinders productive dialogue and ultimately harms the project.
+
+This proposal seeks to establish clear foundations for a shared understanding of moderation practices through the *Memorandum of Understanding on Equitable Moderation* and to ensure these foundations are upheld in practice. It aims to prevent the NixOS foundation mission statement from being misused to enforce a homogenizing moderation policy.
+
+The *Memorandum* should be seen as a normatively binding interpretation of the moderation team's mission statement.
+
+This document aims to articulate an overlapping consensus on reasonable expectations for anyone participating as an equal member in this community, emphasizing that community members are accountable to moderation and vice versa.
+
+# Detailed Design
+[design]: #detailed-design
+
+## Goals
+
+- Adopt the *Memorandum of Understanding on Equitable Moderation* to foster a shared understanding of equitable moderation practices that accommodate a community with diverse sociopolitical views.
+- Ensure transparent accountability of moderation practices to community members.
+- Ground community member accountability in publicly available general principles, providing a stable basis for behavioral expectations.
+- Make wide-ranging decisions regarding community member exclusion understandable.
+- Promote a diverse moderation team to balance diverse perspectives on controversial issues.
+- Establish an *Appeals Council* to hold moderation team members and procedures accountable and to allow for the revision of significant decisions if they fail to meet the community's reasonable standards.
+
+## Memorandum of Understanding on Equitable Moderation
+
+In response to the impression left by past events and with a fervent desire for improvement, we adopt the following principles and maxims as the *Memorandum of Understanding on Equitable Moderation*. These principles serve to clarify the overlapping consensus on reasonable expectations for community participation:
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
```

Comment:
This is nonsensical. Moderation decisions are based on whatever Code of Conduct or similar rules apply, not "general principles". The notion of "concrete reasons that apply equally to all community members, irrespective of characteristics" is vague in the extreme—not to mention the category "sociopolitical orientation".